### DISCUSSION TITLE
Unable to push anything with github-push-block FATAL error

### DISCUSSION THREAD

**Discussion by remiforest** [community/community#135323] [Repositories] [answered]
### Select Topic Area

Question

### Body

Since few days I'm unable to push anyting.
It fails with the following message: 
FATAL: All public push activity has been blocked until further notice, see go/github-push-block for details.

At first I thought it was related to some kind of secrets detected in the code base, but even with a brand new empty repo it fails.
FATAL: Refusing to allow push to remote 'origin' (https://github.com/remiforest/testpush.git)!
FATAL: All public push activity has been blocked until further notice, see go/github-push-block for details.
error: failed to push some refs to 'https://github.com/remiforest/testpush.git'

What can be wrong?

**Comment by jvcByte** [comment, marked as answer]
Hi there @remiforest, the repository you are pushing to doesn't exist in your GitHub account:
![Screenshot 2024-08-10 123119](https://github.com/user-attachments/assets/ff03d80b-f1f6-43e6-8799-2fbade41fd3b)

I suggest you:
* Create the repo and try pushing again
* Check if you still have push permissions on other repositories.
* Remove the secrets from the files(e.g. API keys etc.).

If the issue persist:
* Reach out to [GitHub Support](https://support.github.com/) for further clarification on why your public push activity might be blocked.

PS: You can check [here](https://docs.github.com/en/code-security/secret-scanning/working-with-secret-scanning-and-push-protection) for more info concerning GitHub push protection and code security.

**Reply by remiforest** [reply]
It's a private repo.
However I regenerated the auth token and reset the full Git config, and it fixed the issue.
Thanks!

**Reply by jvcByte** [reply]
It's a pleasure
